mantra/core/plugins/errors.py

```diff
@@ -0,0 +1,89 @@
+import math
+import re
+
+import hikari
+import lightbulb
+
+errors = lightbulb.Plugin("Errors", "Error handling plugin for the bot")
+
+
+@errors.listener(lightbulb.CommandErrorEvent)
+async def on_error(event: lightbulb.CommandErrorEvent):
+    error = event.exception
+
+    if isinstance(error, lightbulb.CommandNotFound):
+        return
+
+    if isinstance(error, lightbulb.BotMissingRequiredPermission):
+        missing = [
+            perm.replace("_", " ").replace("guild", "server").title()
+            for perm in str(error.missing_perms).split("|")
+        ]
+        if len(missing) > 2:
+            fmt = "{}, and {}".format("**, **".join(missing[:-1], missing[-1]))
+        else:
+            fmt = " and ".join(missing)
+
+        description = f"I am missing the **{fmt}** permission(s) to run this command."
+
+        return await event.context.respond(
+            flags=hikari.MessageFlag.EPHEMERAL,
+            embed=hikari.Embed(
+                title="Missing Required Permissions",
+                color=0x00FF00,
+                description=description,
+            ),
+        )
+
+    if isinstance(error, lightbulb.CommandIsOnCooldown):
+        return await event.context.respond(
+            embed=hikari.Embed(
+                title="Command on Cooldown",
+                color=0x00FF00,
+                description=f"This command is on cooldown, try again after `{math.ceil(error.retry_after)}s`.",
+            ),
+            flags=hikari.MessageFlag.EPHEMERAL,
+        )
+
+    if isinstance(error, lightbulb.NotEnoughArguments):
+        return await event.bot.help_command.send_command_help(
+            event.context, event.context.command
+        )
+
+    if isinstance(error, lightbulb.MissingRequiredPermission):
```

Comment:
The `lightbulb.MissingRequiredPermission` and `lightbulb.BotMissingRequiredPermission` are exactly the same.
Can we combine those two and make them one?